avm/res/sql/server/main.bicep

```diff
@@ -685,7 +704,7 @@ type auditSettingsType = {
   retentionDays: int?
 
   @description('Required. Specifies the state of the audit. If state is Enabled, storageEndpoint or isAzureMonitorTargetEnabled are required.')
-  state: 'Enabled' | 'Disabled'
+  state: 'Enabled' | 'Disabled'?
```

Comment:
Optional/Required?

Reply:
IMO required. It user makes conscious choise to configure audit logging then they should also select if logging is enabled or disabled.

Reply:
Optional, default is 'Enabled'